README.md

```diff
@@ -1,2 +1,113 @@
-# FrendsTaskTemplate
-Template for creating Frends Tasks.
+# Frends Task Template
+
+![](https://github.com/FrendsPlatform/FrendsTasksTemplate/workflows/AutoBuildMAster/badge.svg) ![NuGet](https://img.shields.io/nuget/v/frendstask) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+This template can be used to create new Official Tasks for [Frends](frends.com) integration platform. This enables you to start writing code without any hassle with project formats etc.
+
+The created C# solution will use the new tool chain (dotnet new etc.) from .NET Core SDK to ensure cross-platform compatibility. 
+
+You can learn more about custom Tasks [here](https://docs.frends.com/en/articles/2206746-custom-tasks).
+
+## Prerequisite
+
+This template is based on .NET Core CLI tools, so you need to have .NET Core 6.0 SDK or newer. As it is having reaching end of support in 2021-08-21 it is preferably to use [.NET Core 6.0 SDK]().https://dotnet.microsoft.com/en-us/download/dotnet/6.0
+
+Frends tasks are mainly coded in C#, so you will also need a developing environment for it. Visual Studio, Visual Studio Code, and Rider the most widely used ones, but any text editor will do as you can compile from [console](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build). If you are using Visual Studio you can also use, to develop frends Task, old Non-SDK style project files, and MSBuild and nuget.exe technologies. This way you would not need .NET Core 2.1 SDK, but then you are not able to create cross-platform tasks, and creating that kind of C# projects is not supported by this template.
+
+Unfortunately, Visual Studio or other IDEs do not currently support dotnet new templates, so you need to use the command line to install the template and create new Tasks based on it.
+
+## Clone repository
+
+First you need to clone the repository for the new Task. After cloning move to that folder.
+
+## Install the template
+
+You can install the template by following the command.
+
+`dotnet new -i frendstasktemplate --nuget-source https://pkgs.dev.azure.com/frends-platform/frends-tasks/_packaging/main/nuget/v3/index.json` 
+
+## Create a new custom task
+
+You can create a new Task by the following command.
+
+`dotnet new frendstasktemplate --name namespaceForTask --className classNameForTask --taskName TaskName --workflows TaskName`
+
+## Move workflows and .gitignore 
+
+Move workflows from the Task's folder to the .github directory inside the main directory.
```

Comment:
So, can we actually make this run from repository root and include the workflows folder in the template, so we would not need to copy workflows manually?

Reply:
And if not, can we do a post script as described here? https://github.com/dotnet/templating/wiki/Post-Action-Registry#run-script

Reply:
Working on this one.